tests/celery_tests.py

```diff
@@ -219,33 +219,33 @@ def test_run_sync_query_cta_no_data(self):
         query3 = self.get_query_by_id(result3["query"]["serverId"])
         self.assertEqual(QueryStatus.SUCCESS, query3.status)
 
-    def drop_table_if_exists(self, table_name, table_type: CtasMethod, database=None):
+    def drop_table_if_exists(
+        self, table_name, table_type: CtasMethod, database=None
```

Comment:
nit: I know we're far from mypy compliant on our tests, but I try to introduce types here whenever touching these files.
```suggestion
        self, table_name: str, table_type: CtasMethod, database: Optional[Database] = None
```